test/NewsService.Mappers.UnitTests/ResponsesMappers/NewsResponseMapperTests.cs

```diff
@@ -0,0 +1,192 @@
+                Id = Guid.NewGuid(),
+                Content = "Content111",
+                Subject = "Subject111",
+                Author = user,
+                Sender = user,
+                CreatedAt = DateTime.UtcNow
+            };
+
+            dbNews = new DbNews
+            {
+                Id = response.Id,
+                Content = "Content",
+                Subject = "Subject",
+                AuthorName = "Pseudonym",
+                AuthorId = user.Id,
+                SenderId = user.Id,
+                CreatedAt = DateTime.UtcNow,
+                IsActive = true
+            };
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _responseMock = new Mock<IOperationResult<IGetUserDataResponse>>();
+
+            _responseMock
+                .Setup(x => x.Body.FirstName)
+                .Returns(firstName);
+            _responseMock
+                .Setup(x => x.Body.LastName)
+                .Returns(lastName);
+            _responseMock
+                .Setup(x => x.IsSuccess)
+                .Returns(true);
+            _responseMock
+                .Setup(x => x.Body.MiddleName)
+                .Returns(middleName);
+
+            _brokerResponseAuthorNameMock = new Mock<Response<IOperationResult<IGetUserDataResponse>>>();
+            _brokerResponseAuthorNameMock
+                .Setup(x => x.Message)
+                .Returns(_responseMock.Object);
+
+            _brokerResponseSenderNameMock = new Mock<Response<IOperationResult<IGetUserDataResponse>>>();
+            _brokerResponseSenderNameMock
+                .Setup(x => x.Message)
+                .Returns(_responseMock.Object);
+
+            _requestClientMock = new Mock<IRequestClient<IGetUserDataRequest>>();
+
+            _requestClientMock
+                .SetupSequence(x => x.GetResponse<IOperationResult<IGetUserDataResponse>>(
+                    It.IsAny<object>(), default, default))
+                .Returns(Task.FromResult(_brokerResponseSenderNameMock.Object))
+                .Returns(Task.FromResult(_brokerResponseAuthorNameMock.Object));
+
+            _loggerMock = new Mock<ILogger<NewsResponseMapper>>();
+            _mapper = new NewsResponseMapper(_requestClientMock.Object, _loggerMock.Object);
+        }
+
+        [Test]
+        public void ShouldThrowBadRequestExceptionWhenDbNewsIsNull()
+        {
+            Assert.Throws<BadRequestException>(() => _mapper.Map(null));
+        }
+
+        [Test]
+        public void ShouldReturnNewsResponseModelWhenMappingValidDbNews()
+        {
+            var resultNewsModel = _mapper.Map(dbNews);
+            var expectedResult = new NewsResponse();
+
+            expectedResult.Id = dbNews.Id;
+            expectedResult.Subject = dbNews.Subject;
+            expectedResult.CreatedAt = dbNews.CreatedAt;
+            expectedResult.Content = dbNews.Content;
+            expectedResult.Sender = user;
+            expectedResult.Author = user;
```

Comment:
There is no uniform style in object creation (see next test)

Reply:
done